evalml/tests/automl_tests/test_automl.py

```diff
@@ -91,8 +98,8 @@ def test_search_results(X_y_regression, X_y_binary, X_y_multi, automl_type):
         assert isinstance(results['cv_data'], list)
         for cv_result in results['cv_data']:
             assert cv_result.keys() == expected_cv_data_keys
-            if automl_type == ProblemTypes.BINARY:
-                assert isinstance(cv_result['binary_classification_threshold'], float)
+            if objective == 'F1':
+                assert cv_result['binary_classification_threshold'] == 0.5
```

Comment:
Hmm, I am forgetting why this should be `0.5` and not `None`

Reply:
The way that `tune_binary_threshold` is currently defined, if `is_binary(problem_type) and objective.is_defined_for_problem_type(problem_type) and objective.can_optimize_threshold` (true for F1 objective), the pipeline will automatically set its threshold to 0.5.

Reply:
Ah thanks, so according to our current definition, this expected value is correct. 👍